packages/amplication-server/src/core/git/git.provider.service.ts

```diff
@@ -411,6 +420,61 @@ export class GitProviderService {
     });
   }
 
+  async updateProviderOrganizationProperties(
+    gitOrganization: GitOrganization,
+    gitProviderArgs: GitProviderArgs
+  ): Promise<GitProviderArgs> {
+    const { id, installationId, provider, providerProperties } =
+      gitOrganization;
+    const providerPropertiesObj = JSON.parse(
+      JSON.stringify(providerProperties)
+    );
+
+    const currentTime = Math.floor(Date.now() / 1000);
+    const expirationTime = providerPropertiesObj.expiresIn;
```

Comment:
Check since you are comparing seconds since epoch time against a `providerPropertiesObj.expiresIn` that I'm unsure what it represents